web/src/main/java/org/openmrs/web/filter/update/UpdateFilter.java

```diff
@@ -480,6 +481,13 @@ protected boolean isSuperUser(Connection connection, Integer userId) throws SQLE
 	 * @see Listener#startOpenmrs(ServletContext)
 	 */
 	private void startOpenmrs(ServletContext servletContext) throws Exception {
+		
+		// Update the filter to set the Openmrs classloader on the current thread before
+		// starting OpenMRS
```

Comment:
Why a 4 line comment?

Reply:
i have created a new pr https://github.com/openmrs/openmrs-core/pull/3387 in which i have addressed your comments...thanks